src/T2DMap.cpp

```diff
@@ -78,13 +95,235 @@ const QString& key_icon_line_dashDotDot = qsl(":/icons/dash-dot-dot-line.png");
 const QString& key_dialog_ok_apply = qsl("dialog-ok-apply");
 const QString& key_dialog_cancel = qsl("dialog-cancel");
 
+void T2DMap::registerInteractionHandler(IInteractionHandler* handler, int priority)
```

Comment:
Think it would be good to document here how priority works (is higher or lower better)

Reply:
Sure, adding